.pre-commit-config.yaml

```
@@ -20,6 +24,11 @@ repos:
           "--skip=*.csv"
         ]
 
+  - repo: https://github.com/fsfe/reuse-tool
+    rev: v5.1.1
+    hooks:
+      - id: reuse-lint-file
```

FYI only, this will likely only lint on the changed files in a commit. To always check the whole repo (what we do usually):
```suggestion
      - id: reuse
```